Remotion/Data/DomainObjects.ObjectBinding.UnitTests/BindableDomainObjectMixinTests/GetDisplayNameTest.cs

```diff
@@ -38,8 +39,11 @@ public override void SetUp ()
 
       _bindablePropertyReadAccessStrategyMock = new Mock<IBindablePropertyReadAccessStrategy>(MockBehavior.Strict);
 
```

Comment:
I think we should really think about how this should be handled in the future. I dont think we should fix this completely here, as more changes to IoC just make this problem worse, and those will possibly happen at some point. Thoughts?

Reply:
introduce a fake obect similiar to ISecurityProvider and set it up in the setup fixture? 

and yes, eventually we need a better way to do test-specific IoC overrides.

Evnetually, ClientTransaction.CreateRootTransaction will (should) get an IoC-based version and we then have the option of instantiating our own as needed.

Reply:
Well, I'll let you have a look at the different changes and we'll talk then :)